driver/util/odbc_helper.cpp

```diff
@@ -14,52 +14,105 @@
 
 #include "odbc_helper.h"
 
-#ifdef WIN32
-    #include <windows.h>
-#endif
+#include "../odbcapi_rds_helper.h"
 
-#include <sql.h>
-#include <sqlext.h>
+// codechecker_suppress [readability-convert-member-functions-to-static]
+void OdbcHelper::Disconnect(const DBC* &dbc, const std::shared_ptr<RdsLibLoader> &lib_loader) {
+    if (dbc->wrapped_dbc) {
```

Comment:
Just so we're extra safe, let's also validate that dbc itself is valid
```suggestion
    if (dbc && dbc->wrapped_dbc) {
```